File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.py
angles = sorted(list(int(input()) for _ in range(3)))
if sum(angles) == 180:
    if angles[0] == angles[1] == angles[2]:
        print("Equilateral")
    elif angles[0] == angles[1] or angles[1] == angles[2]:
        print("Isosceles")
    else:
        print("Scalene")
else:
    print("Error")

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.cc
#include <iostream>
using namespace std;

int main() {
	cin.tie(0)->sync_with_stdio(0);

	int a, b, c;
	cin >> a >> b >> c;
	if (a + b + c != 180) cout << "Error";
	else if (a == 60 && b == 60 && c == 60) cout << "Equilateral";
	else if (a == b || b == c || c == a) cout << "Isosceles";
	else cout << "Scalene";
}